src/main/blackbox/blackbox_fielddefs.h

```diff
@@ -54,6 +59,22 @@ typedef enum FlightLogFieldCondition {
     FLIGHT_LOG_FIELD_CONDITION_LAST = FLIGHT_LOG_FIELD_CONDITION_NEVER
 } FlightLogFieldCondition;
 
+typedef enum SelectField { // no more than 16
+    SELECT_FIELD_PID = 0,
+    SELECT_FIELD_RC_COMMANDS,
+    SELECT_FIELD_SETPOINT,
+    SELECT_FIELD_BATTERY,
+    SELECT_FIELD_MAG,
+    SELECT_FIELD_ALTITUDE,
+    SELECT_FIELD_RSSI,
+    SELECT_FIELD_GYRO,
+    SELECT_FIELD_ACC,
+    SELECT_FIELD_DEBUG,
+    SELECT_FIELD_MOTOR,
+
+    SELECT_FIELD_GPS = 15
```

Comment:
No reason to artificially put GPS at the end.

Reply:
Put at the end to don't interfere with possibly new fields. So it's not a problem if it's between the other fields?

Reply:
This is just an enumeration of bit positions. Why would the ordering of the fields matter at all? All that matters is that they don't change and we append new ones to the end in the future.

Reply:
Ok, perfect!